src/main/java/at/hannibal2/skyhanni/tweaker/DownloadSourceChecker.java

```diff
@@ -94,14 +96,15 @@ private static void openMenu(URI host) {
             }
         ));
 
+        String version = SkyHanniMod.Companion.getVersion();
         JOptionPane.showOptionDialog(
             frame,
             String.format(String.join("\n", SECURITY_POPUP), uriToSimpleString(host)),
-            "SkyHanni Security Error",
+            "SkyHanni " + version + " Security Error",
             JOptionPane.DEFAULT_OPTION,
             JOptionPane.ERROR_MESSAGE,
             null,
-            new JPanel[] { links, buttons },
+            new JPanel[]{links, buttons},
```

Comment:
?

Reply:
auto formatting for me. is this wrong? idk